templates/new/config/host.exs

```diff
@@ -1,3 +1,22 @@
 import Config
 
 # Add configuration that is only needed when running on the host here.
+
+config :nerves_runtime,
+  kv_backend:
+    {Nerves.Runtime.KVBackend.InMemory,
+     contents: %{
+       # The KV store on Nerves systems is typically read from UBoot-env, but
+       # this allows us to use a pre-populated InMemory store when running on
+       # host for development and testing.
+       #
+       # https://hexdocs.pm/nerves_runtime/readme.html#using-nerves_runtime-in-tests
+       # https://hexdocs.pm/nerves_runtime/readme.html#nerves-system-and-firmware-metadata
+
+       "nerves_fw_active" => "a",
+       "a.nerves_fw_architecture" =>
+         :erlang.system_info(:system_architecture) |> to_string() |> String.split("-") |> hd(),
+       "a.nerves_fw_description" => "Host KV store",
+       "a.nerves_fw_platform" => "host",
+       "a.nerves_fw_version" => "0.1.0"
```

Comment:
```suggestion
       "a.nerves_fw_architecture" => "generic",
       "a.nerves_fw_description" => "N/A",
       "a.nerves_fw_platform" => "host",
       "a.nerves_fw_version" => "0.0.0"
```
I dumbed down the defaults. For architecture, I wanted to make sure that if this made it way to being checked in a unit test that it would be deterministic. For version, I didn't want it accidentally matching someone's firmware version and leaving the impression that it was real.